profiles/Dartellum-setup.yaml

```diff
@@ -1155,31 +1199,34 @@ empty_hunting_room_messages:
 
 hunting_buddies_max: 3
 hunting_buddies:
- - Ceindrech
- - Treahead
- - Anshelm
- - Haruhiko
- - Iney
- - Endraig
- - Reverence
- - Twiddlethumbs
- - Vinarterta
- #- Crannach
- - Aurayn
- - Gallifreius
- - Sarvatt
- - Atanamir
- - Ezzra
- - Mitkiahn
- - Venantius
- - Gram
- - Emma
- - Kalinandra
- - Emmajin
- - Wilhellm
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ #- xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+ - xxxxx
+
+ combat_teaching_skill: Parry
```

Comment:
❌ There is a leading space before this setting that is breaking the YAML parser

Reply:
Fixed.